code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
And yet half the server didn't even know what the PR the poll was asking them about really did. That's not a good design for a poll.